Cargo.toml

```diff
@@ -13,12 +13,21 @@ build = "build.rs"
 [lib]
 name = "rls"
 doctest = false
+path = "rls/src/lib.rs"
 
 [[bin]]
 name = "rls"
 test = false
+path = "rls/src/main.rs"
 
 [dependencies]
+rls-analysis = "0.16.12"
```

Comment:
I thought it might make it more obvious as to what's happening here but maybe there's no actual benefit in doing that?